doc/nrf/samples/wifi_zephyr.rst

```diff
@@ -27,42 +24,26 @@ The following list specifies samples that are currently supported with the Wi-Fi
 * :zephyr:code-sample:`syslog-net`
 * :zephyr:code-sample:`telnet-console`
 
-Configuration
-*************
-
-|config|
-
-Before you build a sample, you must configure the following Wi-Fi credentials in the :file:`overlay-nrf700x.conf` file:
-
-* :kconfig:option:`CONFIG_WIFI_CREDENTIALS_STATIC_SSID` - Network name (SSID)
-* :kconfig:option:`CONFIG_WIFI_CREDENTIALS_STATIC_PASSWORD` - Password
-* :kconfig:option:`CONFIG_WIFI_CREDENTIALS_STATIC_TYPE` - Security type (Optional)
-
-.. note::
-   You can also use ``menuconfig`` to configure Wi-Fi credentials.
-
-See :ref:`zephyr:menuconfig` in the Zephyr documentation for instructions on how to run ``menuconfig``.
-
 Building and running
 ********************
 
 To build the sample with |VSC|, follow the steps listed on the `How to build an application`_ page in the |nRFVSC| documentation.
 See :ref:`building` for other building scenarios, :ref:`programming` for programming steps, and :ref:`testing` for general information about testing and debugging in the |NCS|.
 
-An overlay file, ``overlay-nrf700x.conf`` is provided to all Zephyr samples, which configures the sample to run with the Wi-Fi driver.
+A :ref:`Wi-Fi snippet <zephyr:snippet-wifi-ipv4>` configuration is provided to all Zephyr samples, which configures the sample to run with the Wi-Fi driver.
 
 To build Zephyr samples for the nRF7002 DK, use the ``nrf7002dk/nrf5340/cpuapp`` board target.
 The following is an example of the CLI command:
 
 .. code-block:: console
 
-   west build -b nrf7002dk/nrf5340/cpuapp -- -DEXTRA_CONF_FILE=overlay-nrf700x.conf
+   west build -b nrf7002dk/nrf5340/cpuapp -S wifi-ipv4
 
 To build for the nRF7002 EK with nRF5340 DK, use the ``nrf5340dk/nrf5340/cpuapp`` board target with the ``SHIELD`` CMake option set to ``nrf7002ek``.
 The following is an example of the CLI command:
 
 .. code-block:: console
 
-   west build -b nrf5340dk/nrf5340/cpuapp -- -DSHIELD=nrf7002ek -DEXTRA_CONF_FILE=overlay-nrf700x.conf
+   west build -b nrf5340dk/nrf5340/cpuapp -S wifi-ipv4 -- -DSHIELD=nrf7002ek
 
 For additional details about running a sample, refer to the respective sample in Zephyr’s Samples and Demos documentation.
```

Comment:
Suggesting to add the reference to the Zephyr documentation.
```suggestion
For additional details about running a sample, refer to the respective sample in Zephyr’s :ref:`Samples and Demos documentation <zephyr:samples-and-demos>`.
```

Reply:
Thanks, updated according to the suggestion.